dds/src/transport/types.rs

```diff
@@ -110,28 +140,60 @@ pub struct EntityId {
 }
 
 impl EntityId {
+    #[inline]

```

Comment:
Why do you use [inline] in so many places?

Reply:
Since these methods contain only a single line of code, we hint the compiler to inline them. See [inlining](https://nnethercote.github.io/perf-book/inlining.html) and [inline in Rust](https://matklad.github.io/2021/07/09/inline-in-rust.html).
The latter is considered a good practice, and as library authors, we should apply these practices 🙌